docs/docs/configuration/filtering.md

```diff
@@ -1,8 +1,34 @@
 # Filtering
 Trivy provides various methods for filtering the results.
 
+```mermaid
+flowchart LR
+  Issues("Detected\nIssues") --> Severity
+
+  subgraph Filtering
+    subgraph Prioritization
+        direction TB
+        Severity("By Severity") --> Status("By Status")
+    end
+    subgraph Suppression
+        Status --> Ignore("By Finding IDs")
+        Ignore --> Rego("By Rego")
+        Rego --> VEX("By VEX")
+    end
+  end
+  VEX --> Results
+```
+
+Similar to the functionality of filtering results, you can also limit the sub-targets for each scanner.
+For information on these settings, please refer to the scanner-specific documentation ([vulnerability](../scanner/vulnerability.md) , [misconfiguration](../scanner/misconfiguration/index.md), etc.).
+
+## Prioritization
+You can filter the results by 
+
+- Severity
+- Status
```

Comment:
nit: it will be easier to use if you add a link here (as for https://aquasecurity.github.io/trivy/v0.49/docs/scanner/vulnerability/).

Reply:
Link to the vulnerability page? Or do you mean an anchor?

Reply:
anchors:
```suggestion
- [Severity](#by-severity)
- [Status](#by-status)
```